positional/tests/parse_row.rs

```diff
@@ -35,3 +35,27 @@ fn ser_de() {
         FromPositionalRow::from_positional_row(&row).expect("error converting from positional row");
     assert_eq!(original_data, data);
 }
+
+#[test]
+fn empty_string() {
+    let row = <Data as FromPositionalRow>::from_positional_row("");
```

Comment:
You don't need to qualify the trait here, you can just call `Data::from_positional_row()`.